### ISSUE TITLE
‘org-element-at-point’ cannot be used in non-Org buffer

### ISSUE THREAD

**Issue by tkossak** [syl20bnr/spacemacs#16575]
#### Description :octocat:
When I press `SPC j i` (command `spacemacs/helm-jump-in-buffer`) I get multiple errors:
`⛔ Warning (org-element): ‘org-element-at-point’ cannot be used in non-Org buffer #<buffer *helm org inbuffer*> (helm-major-mode)`

#### Reproduction guide :beetle:
- create file `test.org` with content:
```
* heading1
blabla
```
- you need to also have `(setq helm-org-format-outline-path nil)` in your `dotspacemacs/user-config` in `.spacemacs` file
- Start Emacs and open `test.org` file
- press `SPC j i`

*Observed behaviour:* :eyes: :broken_heart:
- the error appears in separate buffer

*Expected behaviour:* :heart: :smile:
- there should be no error

#### System Info :computer:
- OS: gnu/linux
- Emacs: 29.4
- Spacemacs: 0.999.0
- Spacemacs branch: develop (rev. 2cb35317a)
- Graphic display: t
- Running in daemon: nil
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(emacs-lisp helm org treemacs)
```
- System configuration features: ACL CAIRO DBUS FREETYPE GIF GLIB GMP GNUTLS GPM GSETTINGS HARFBUZZ JPEG JSON LCMS2 LIBOTF LIBSYSTEMD LIBXML2 M17N_FLT MODULES NOTIFY INOTIFY PDUMPER PNG RSVG SECCOMP SOUND SQLITE3 THREADS TIFF TOOLKIT_SCROLL_BARS TREE_SITTER WEBP X11 XDBE XIM XINPUT2 XPM GTK3 ZLIB
```

**Comment by tkossak** [comment]
I found out that the problem is caused by `(setq helm-org-format-outline-path nil)` in my user-config.

**Comment by tkossak** [comment]
@smile13241324 what do you mean "not planned" - the bug won't be fixed?

**Comment by fnussbaum** [comment]
I cannot reproduce this. For me your reproduction steps succeed without errors or warnings for both values of `helm-org-format-outline-path`.  Are your packages up to date?

**Comment by smile13241324** [comment]
Sorry @tkossak I have understood your last comment that this was a user config error. If not I will reopen this issue

**Comment by tkossak** [comment]
I just tried to reproduce it with new spacemacs installation:
```
mv ~/.emacs.d ~/.emacs.d.old
mv ~/.spacemacs ~/.spacemacs.old

git clone --recursive https://github.com/syl20bnr/spacemacs ~/.emacs.d
```
Now copy my sample [dotfile](https://gist.github.com/tkossak/76833f64c27c2aff52a4bbbdd0799717) to `~/.spacemacs`. Only few lines changed from default dotfile - I added org layer, removed multi cursor layer and added these 2 lines:
```
  (setq helm-org-format-outline-path nil)
  (setq org-startup-indented 't)
```

Create sample org file (eg like in first post), start Spacemacs and open the file. Press `SPC j i`, result:
![image](https://github.com/user-attachments/assets/05bfd614-dcb7-4ae3-98c1-826d88816613)

When I comment out the line `(setq helm-org-format-outline-path nil)` then it works without any errors. But it also works when I comment out only the second line `(setq org-startup-indented 't)`, so not sure what is the culprit here.

**Comment by fnussbaum** [comment]
This seems to be a problem upstream in https://github.com/emacs-helm/helm-org, and should be reported there.
As a workaround it should be relatively safe to ignore this warning with something like:
```elisp
  (add-to-list 'warning-suppress-types 'org-element-parser)
```
(These warnings will still get logged to the buffer, but they will not pop up anymore.)

**Comment by tkossak** [comment]
Thank you. I posted it in helm-org github.
The silence trick doesn't work properly for me. I added it to my `spacemacs/user-config` function, and now when I press `SPC j i` I don't get dialog (to jump to specific heading) at all. Instead I get error in status bar:
`Error running timer ‘org-indent-initialize-agent’: (wrong-type-argument listp org-element-parser)`

For now I got rid of the option `(setq org-startup-indented 't)` from my dotfile.
